paddle/fluid/API.spec

```diff
@@ -271,6 +271,10 @@ paddle.fluid.layers.reciprocal ArgSpec(args=['x', 'name'], varargs=None, keyword
 paddle.fluid.layers.square ArgSpec(args=['x', 'name'], varargs=None, keywords=None, defaults=(None,))
 paddle.fluid.layers.softplus ArgSpec(args=['x', 'name'], varargs=None, keywords=None, defaults=(None,))
 paddle.fluid.layers.softsign ArgSpec(args=['x', 'name'], varargs=None, keywords=None, defaults=(None,))
+paddle.fluid.layers.not_equal ArgSpec(args=['x', 'name'], varargs=None, keywords=None, defaults=(None,))
```

Comment:
Why is there only `x`? I thought is should be `x != y`

Reply:
It seems that [generate_layer_fn](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/layers/layer_function_generator.py#L122) is invalid for the operator with multiple inputs. I will check it.